### ISSUE TITLE
LinkedIn posting issues

### ISSUE THREAD

**Issue by instantiator** [instantiator/open-social-distributor#1]
The v0.1 announcement post failed in a couple of interesting ways on LinkedIn:

* The link was not included in the post
* The text cut off in the 3rd stanza, after `🧵 This post` and before `(and thread`

**Comment by instantiator** [comment]
I've added some tests to rule out the fomatter (it's working fine).

**Next step:** Test some variants of the release announcement to figure out if there's an issue in the text.
**To investigate:** Does commentary text need to be encoded?

Here's the code for posting from `LinkedInNetwork`:

```csharp
imagesContent = PrepareContent(imagesData);
var postRequestContent = new 
{ 
    author = author, 
    commentary = text,
    visibility = "PUBLIC",
    distribution = new 
    {
        feedDistribution = "MAIN_FEED",
        targetEntities = new object[] { },
        thirdPartyDistributionChannels = new object[] { }
    },
    content = imagesContent,
    contentLandingPage = message.Link?.ToStringFor(NetworkType) ?? "",
    lifecycleState = "PUBLISHED",
    isReshareDisabledByAuthor = false
};
postRequest.AddJsonBody(postRequestContent);
var postResponse = await client!.ExecuteAsync(postRequest);
```

**Comment by instantiator** [comment]
[This looks a lot like the same issue](https://stackoverflow.com/questions/73712703/linkedin-post-api-post-text-gets-cut-off-if-contains) - text is cut off at brackets (ie. `(` and `)`).

**Comment by instantiator** [comment]
Resolved in release `v0.2`
